bin/Powheg/run_lhe_parallel.sh

```diff
@@ -3,12 +3,21 @@
 fail_exit() { echo "$@"; exit 1; }
 
 #set -o verbose
+<<<<<<< HEAD
```

Comment:
this change has to do with the additional parameter to pass the number of threads.
This script (run_lhe_parallel.sh)  should be used with NNLOPS locally, probably is irrelevant (but better to be consistent and propagate the number of threads)